### DISCUSSION TITLE
Bar / Line Graphing

### DISCUSSION THREAD

**Discussion by Ed-Tronics** [RudolphRiedel/FT800-FT813#120] [General]
Hi All,

Just wondered if anyone in the community has started a graphing project? Either bar graphs or line graphs (drawing value / time in real time). I've got some ideas how to do it but wondered if anyone has produced any particularly ingenious/efficient concepts?

Cheers.

**Comment by RudolphRiedel** [comment]
Nothing really fancy, but one of my earlier projects back in 2017 used a line-strip with 64 points for a scope-like display.
And there also was a bar graph with 16 rectangles.

![FT813_RVT70UQFNWC0x jpg_2](https://github.com/RudolphRiedel/FT800-FT813/assets/31180093/29d9fd9c-758b-4c28-8173-51c4fa76fd5b)
 
https://www.mikrocontroller.net/attachment/324091/FT8xx_Test_90CAN_FT813_RVT70UQFNWC0x_2017-03-18.zip

Woah, given the age I propably should not unpack that archive. :-)

**Reply by Ed-Tronics** [reply]
This has given me some food for thought. Thanks very much!
